src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package calculator;
+
+import java.util.regex.Pattern;
+
+import static java.lang.Integer.parseInt;
+
+class Calculator {
+    private static final Pattern NORMAL_PATTERN = Pattern.compile("^[\\d]+(\\s[\\d+*\\-/])*.\\d$");
+
+    private String operator;
+    private String string;
+
+    private int reduce;
```

Comment:
```suggestion
    private final String operator;
    private final String string;

    private int reduce;
```

소소하지만..! final 키워드를 통해 변하지 않는 값과 변하는 값이라는 의미를 전달할 수 있습니다..ㅎㅎ
정말 소소한 코멘트네요...!!!

Reply:
이부분은 계속 재할당되야 하는 구조라 final로 선언하지 않았습니다. 
의도된 부분이에요 !